.github/workflows/ci.yml

```diff
@@ -287,6 +288,18 @@ jobs:
           target: ${{ matrix.target }}
           image: ${{ steps.build-docker-image.outputs.image }}
 
+      - name: Test Cross Image
+        if: (steps.prepare-meta.outputs.has-image && (github.ref == 'refs/heads/staging' || github.ref == 'refs/heads/trying')) && matrix.target == 'cross'
+        run: ./ci/test-cross-image.sh
+        env:
+          TARGET: 'aarch64-unknown-linux-gnu'
+        shell: bash
+      - uses: ./.github/actions/cargo-install-upload-artifacts
+        if: matrix.deploy
+        with:
+          target: ${{ matrix.target }}
+          image: ${{ steps.build-docker-image.outputs.image }}
```

Comment:
again? See line L285-L289. Those lines should be removed if this is kept

Reply:
Ah my mistake, is this only keeping the artifacts for cross itself? I assumed it was the artifacts for the Docker image, but that's probably my mistake.

Reply:
I removed these lines from `Test Cross Image`. I also didn't add LLVM coverage to this PR, since it makes little sense (everything is running inside a container and with a different target directory/etc).